distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -76,6 +77,12 @@ Text.reverse self =
    Arguments:
    - function: The operation to apply to each character in the text.
 
+   This method does not return the results, so is only useful for performing
```

Comment:
```suggestion
   This method does not return the results, so it is only useful for performing
```

Not 100% sure but it felt like the subject was missing in that sentence

Reply:
fixed